cups/utils.py

```diff
@@ -14,23 +12,30 @@ def _strtobool(val: str) -> Optional[bool]:
     'val' is anything else.
     """
     val = val.lower()
-    if val in ('y', 'yes', 't', 'true', 'on', '1'):
+    if val in ("y", "yes", "t", "true", "on", "1"):
         return True
-    elif val in ('n', 'no', 'f', 'false', 'off', '0'):
+    elif val in ("n", "no", "f", "false", "off", "0"):
         return False
     else:
         return None
 
+
 def _bytes_to_value(b: bytes) -> Optional[Any]:
```

Comment:
Typing would be improved if you did
```suggestion
def _bytes_to_value(b: bytes) -> Optional[Union[str, bool]]:
```